apps/functions-subscription/src/adapters/azure/functions/codec.ts

```diff
@@ -56,3 +58,23 @@ export const toTrialAPI = (trial: Trial): TrialAPI => {
       }
     : trialAPI;
 };
+
+export const toSlimTrialAPI = (trial: Trial): TrialSlim => {
```

Comment:
**suggestion**: what do you think?

just to be coherent with other naming
```suggestion
export const toTrialSlimAPI = (trial: Trial): TrialSlim => {
```

Reply:
Renamed in ce70af07d12fba443dabfbc5aa2b2a899a27949b